sys/net/network_layer/ipv6/addr/ipv6_addr.c

```diff
@@ -106,6 +111,32 @@ void ipv6_addr_init_iid(ipv6_addr_t *out, const uint8_t *iid, uint8_t bits)
     memcpy(&(out->u8[pos]), iid, bytes);
 }
 
+int ipv6_addr_split(char *addr_str, char seperator, int _default)
+{
+    char *sep = addr_str;
+    while(*++sep) {
+        if (*sep == seperator) {
+            *sep++ = '\0';
+            if (*sep) {
+                _default = atoi(sep);
+            }
+            break;
+        }
+    }
+
+    return _default;
+}
+
+#ifdef MODULE_FMT
+void print_ipv6_addr(const ipv6_addr_t *addr)
```

Comment:
>Well, this PR just adds the functions (which I needed in other branches), and to #4754 I just added the interface selection. I didn't want to change all the prints in order to keep the PR clean.

:+1: 

another btw, please add a `[ci skip]` to your commit messages that are intended to be squashed. The Travis queue overloads otherwise with several build attempts for obsolete commit ids.